packages/renderer/src/lib/pod/PodmanPodDetailsSummary.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+import { Link } from '@podman-desktop/ui-svelte';
+import { router } from 'tinro';
+
+import DetailsCell from '../details/DetailsCell.svelte';
+import DetailsSubtitle from '../details/DetailsSubtitle.svelte';
+import DetailsTable from '../details/DetailsTable.svelte';
+import DetailsTitle from '../details/DetailsTitle.svelte';
+import type { PodInfoUI } from './PodInfoUI';
+
+export let pod: PodInfoUI;
+let creationTime: Date = new Date(pod.created);
+</script>
+
+<div class="flex px-5 py-4 flex-col items-start h-full overflow-auto">
+  {#if pod}
+    <DetailsTable>
+      <tr>
+        <DetailsTitle>Details</DetailsTitle>
+      </tr>
+      <tr>
+        <DetailsCell>Name</DetailsCell>
+        <DetailsCell>{pod.name}</DetailsCell>
+      </tr>
+      <tr>
+        <DetailsCell>ID</DetailsCell>
+        <DetailsCell>{pod.id}</DetailsCell>
+      </tr>
+      <tr>
+        <DetailsCell>Created</DetailsCell>
+        <DetailsCell>{creationTime}</DetailsCell>
+      </tr>
+      <tr>
+        <DetailsCell>Age</DetailsCell>
+        <DetailsCell>{pod.age}</DetailsCell>
+      </tr>
+      {#if pod}
+        <tr>
+          <DetailsTitle>Pod Status</DetailsTitle>
+        </tr>
+        <tr>
+          <DetailsCell>Status</DetailsCell>
+          <DetailsCell>{pod.status.toLowerCase()}</DetailsCell>
+        </tr>
+        {#if pod.containers.some(containerStatus => containerStatus.Status)}
+          <tr>
+            <DetailsTitle>Containers</DetailsTitle>
+          </tr>
+          {#each pod.containers as containerStatus}
```

Comment:
Can we rename `containerStatus` to maybe `container` or `containerUI`? Had to take a second look below b/c I wasn't sure it was referring to the container.